src/api/axiosInstance.js

```diff
@@ -19,10 +23,25 @@ const requestClient = () => {
   let instance = axios.create(defaultOptions);
   instance.interceptors.response.use((response) => {
     return response;
-  }, (error) => {
-    console.log("ERROR RESPONSE");
-  });
+  }, errorHandler);
+  return instance;
+}
+
+const requestClientV1 = () => {
```

Comment:
Rename to `requestExplorerServiceV1(...)`?

Reply:
Sure, make sense after renaming the constant